npm-package/src/tests/mock app/mock-app.ts

```diff
@@ -0,0 +1,40 @@
+import express, {
```

Comment:
See earlier comments about test folder being separate from source code folder.

Could you also elaborate on the usage of these files in the mock app folder? They seem to be exact copies of the actual server. Could we use the actual server and router for the tests?